.config/bashrc

```diff
@@ -0,0 +1 @@
+eval "$(direnv hook bash)"
```

Comment:
While at it, please add a comment here as well on top, that the whole thing is for replit support.